e3db/client.py

```diff
@@ -1510,8 +1510,47 @@ def __write_note(self, data, options: NoteOptions, recipient_signing_key, signin
         # Assign signature from response to Note here
         return response.status_code
 
+    def read_note(self, note_id, auth_params={}, auth_headers={}) -> Note:
+        """
+        Pubic method to make a read a note by note_id.
+
+        Parameters
+        ----------
+        note_id : str
+            UUID assigned by Tozstore, used to identify a note.
+
+        auth_params : dict
+            Extra request parameters for EACP authorizations. Optional.
+
+        auth_headers : dict
+            Extra request headers for EACP authorizations. Optional
+
+        Returns
+        -------
+        e3db.Note
+            Decrypted E3DB note
+
+        """
+        if self.public_signing_key == "":
+            raise RuntimeError("Cannot read notes without a signing key!")
+        auth_params['note_id'] = note_id
+        return self.__read_note(auth_params, auth_headers)
+
+    def __read_note(self, params: dict, headers: dict):
+        url = self.__get_url("v2", "storage", "notes")
+        
+        # Is there a case in which client has signing keys but no tsv1_auth, or vice versa? 
+        # Could error here if passing in uninstantiated auth. 
+        response = requests.get(url=url, auth=self.e3db_tsv1_auth, params=params, headers=headers)
+        self.__response_check(response)
+        note = Note.decode(response.json())
+        decrypted_note = self.decrypt_note(note, self.encryption_keys.private_key)
+        note = Note.decode(decrypted_note.to_json())
```

Comment:
decrypt_note is typed as returning a `Note` object, can we return that directly?

Reply:
Yup, we can. Removed the decoding of `decrypted_note`